actors/builtin/market/market_test.go

```diff
@@ -566,6 +876,24 @@ func (h *marketActorTestHarness) getDealState(rt *mock.Runtime, dealID abi.DealI
 	return s
 }
 
+func (h *marketActorTestHarness) dealsByEpochShouldContain(rt *mock.Runtime, epoch abi.ChainEpoch, dealId abi.DealID) {
```

Comment:
I'd prefer not to do this kind of detailed state inspection. We should instead test that deals are processed correctly over time by exercising the passage of time, not checking in here. We should be able to change this impl a lot without breaking tests.

For now, I suggest omitting this and coming back to test deal lifecycle (which will exercise this)

Reply:
I agree. I have removed this.